mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -1573,7 +1585,7 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             };
 
             info!(
-                "Sent taker funding spend (taker payment) {} tx {:02x} during swap {}",
+                "Sent taker funding spend (taker payment spend) {} tx {:02x} during swap {}",
```

Comment:
Then its better to use only `"Sent taker payment {} tx {:02x} during swap {}"` to avoid confusion
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2261/commits/5bd4ee2a4fa2b6fa51a0690d50d0310c1d1ea213